src/index.js

```diff
@@ -131,8 +131,8 @@ class Service {
 
     let { query, options } = multiOptions(id, params, this.id);
 
-    // We can not update the id
-    delete data[this.id];
```

Comment:
i.e. We need to answer the question "Can a primary key be changed during a `put`/`patch`?" That was the reason for the logic I wrote; I assumed the answer to that question is no. But if yes, then we should also allow `_id` to be changed, if not, then we should always ignore what is passed in the body.